salt/aws/init.sls

```diff
@@ -0,0 +1,24 @@
+{% from 'lib.sls' import set_config %}
+
+python3-pip:
+  pkg.installed:
+    - name: python3-pip
+
+awscli:
+  pip.installed:
+    - name: awscli
+    - require:
+      - pkg: python3-pip
+
+/home/sysadmin-tools/aws-settings.local:
+  file.managed:
+    - source: salt://aws/files/aws-settings.local
+    - mode: 640
+    - replace: False
+    - require:
+      - file: /home/sysadmin-tools/bin
```

Comment:
Agreed, and added `require` `file: /home/sysadmin-tools/bin`.

Better than this would be requiring `/home/sysadmin-tools/{{ filename }}` however this errors as a recursive requisite because of how salt loads it.